File: Programming Basics with Python/06 Nested loops/More exercises/09_Sum_of_Two_Numbers.py
interval_start = int(input())
interval_end = int(input())
magick_number = int(input())
counter = 0
breaker = False
for number_one in range(interval_start, interval_end + 1):
    for number_two in range(interval_start, interval_end + 1):
        counter += 1
        if number_one + number_two == magick_number:
            sum = number_one + number_two
            print(f"Combination N:{counter} ({number_one} + {number_two} = {sum})")
            breaker = True
            break
    if breaker:
        break
if not breaker:
    print(f"{counter} combinations - neither equals {magick_number}")

File: JS Basics/06 Nested Loops - More Exercises/09. Sum of Two Numbers.js
function sumOfTwo(input) {
    let startInterval = Number(input[0]);
    let endInterval = Number(input[1]);
    let magickNumber = Number(input[2]);
    let combination = 0;
    let combinationFlag = false;
    for (let firstNumber = startInterval; firstNumber <= endInterval; firstNumber++) {
        for (let secondNumber = startInterval; secondNumber <= endInterval; secondNumber++) {
            combination++;
            if (firstNumber + secondNumber == magickNumber) {
                combinationFlag = true;
                console.log(`Combination N:${combination} (${firstNumber} + ${secondNumber} = ${magickNumber})`);
                break;
            }
        }
        if (combinationFlag == true) {
            break;
        }
    }
    if (combinationFlag == false) {
        console.log(`${combination} combinations - neither equals ${magickNumber}`);
    }
}
sumOfTwo([88, 888, 2000])